src/GameSession.js

```diff
@@ -58,6 +58,16 @@ class GameSession {
     handlePlayerInput(player, input) {
         if (input.type === 'click') {
             this.handleClick(player, input.data);
+        } else if (input.type === 'onHold') {
+            const node = this.findNode(input.nodeId);
+            if (node && node.node.holdHandlers) {
+                node.node.holdHandlers.onHold && node.node.holdHandlers.onHold(player, input.data)
```

Comment:
todo: remove click find logic in game session since client knows id of what it clicked